helm/kubernetes-coredns-chart/values.yaml

```diff
@@ -38,6 +38,10 @@ cluster:
       IP: 172.31.0.0
     domain: cluster.local
 
+loadbalance:
+  policy: round_robin
+
+
```

Comment:
```suggestion
```

Reply:
Also helm suggest flat values, especially if there is only one. I.e. using `loadbalancePolicy` here. But not critical: 
https://docs.helm.sh/chart_best_practices/#flat-or-nested-values